### PR TITLE
Replaced community.kubernetes with kubernetes.core in the openshift4_aws workshop

### PR DESCRIPTION
**Pull request by tarexveff**
The community.kubernetes Ansible collection has been hard-deprecated in favor of kubernetes.core. This pull request replaces all occurrences of community.kubernetes with the recommended kubernetes.core replacement within the openshift4_aws workshop directory path.

### CODE REVIEW

**Comment by ajacocks** [comment]
Thanks! We're working on this across all the module names, in all the workshops.

**Comment by dpullman-emergent** [comment]
Hello! Is there anything we can do to get the two PRs, this one #183 and #181 merged? It may be there is some overlap? We've been running with local fixes in our fork waiting for this to merge, in particular the kubernetes.core FQCN changes.

Thanks!

**Comment by ajacocks** [comment]
Sorry, there aren't as many folks using these, since the original support team was blown up, 2 years ago. We can try to see if we can get it done. But, we could use someone who wants to be a new maintainer.

**Review by ajacocks — APPROVED** [review]
Let me know if this breaks anything

**Pull request merged**
